### PR TITLE
[ADD] Condition count and interval must be greater than 0

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_resource_calendar/models/calendar_event.py [hunk 1]
```diff
@@ -160,6 +160,19 @@ def _get_weekday_number(self):
                 self.start_datetime, '%Y-%m-%d %H:%M:%S'
             ).weekday()
 
+    @api.constrains('interval')
+    def _check_interval_greater_than_0(self):
+        for record in self:
+            if record.interval < 1:
+                raise ValidationError(_('The interval must be greater than 0'))
```
Comment (rimbendhaou) [inline project_resource_calendar/models/calendar_event.py:172]: add translation

File: project_resource_calendar/models/calendar_event.py [hunk 2]
```diff
@@ -160,6 +160,19 @@ def _get_weekday_number(self):
...
+    
+    @api.constrains('count')
+    def _check_count_greater_than_0(self):
+        for record in self:
+            if record.count < 1:
+                raise ValidationError(
+                    _('The number of repetitions must be greater than 0'))
```
Comment (rimbendhaou) [inline project_resource_calendar/models/calendar_event.py:179]: add translation

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_resource_calendar/models/calendar_event.py (hunk 1 above)
Comment (eilst) [reply project_resource_calendar/models/calendar_event.py:172]: done

File: project_resource_calendar/models/calendar_event.py (hunk 2 above)
Comment (eilst) [reply project_resource_calendar/models/calendar_event.py:179]: done

**Comment by eilst** [comment]
Detached HEAD problems, new PR: https://github.com/savoirfairelinux/project-addons/pull/433

**Pull request closed without merging**